server/tests/passwordResetTest.test.js

```diff
@@ -0,0 +1,90 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+const router = () => chai.request(app);
+
+describe('Test sending password reset endpoint', () => {
+    const user = { 
+        email: "aggyreina@gmail.com",
+        password: "aggggg", 
+        email1: "aggy@gmail.com", 
+        password1: "222",
+        email2: "aggyre44mail.com",
+        password2: "aaa",
+        email3: "",
+        password3: ""
+    };
+
+    it('should send a reset link', (done) => {
+        router()
+            .post('/api/v1/reset-link')
+            .send({ email: user.email })
+            .end((err, res) => { 
+                expect(res.body.status).to.be.equal(200);
```

Comment:
Consider testing the message property as well. You should test all properties of the object response.

Reply:
Done